ee/api/test/test_organization.py

```diff
@@ -33,17 +33,8 @@ def test_no_delete_last_organization(self):
         org_id = self.organization.id
         self.assertTrue(Organization.objects.filter(id=org_id).exists())
         response = self.client.delete(f"/api/organizations/{org_id}")
-        self.assertEqual(
-            response.data,
-            {
-                "attr": None,
-                "detail": f"Cannot remove organization since that would leave member {self.CONFIG_USER_EMAIL} organization-less, which is not supported yet.",
-                "code": "invalid_input",
-                "type": "validation_error",
-            },
-        )
-        self.assertEqual(response.status_code, 400)
-        self.assertTrue(Organization.objects.filter(id=org_id).exists())
+        self.assertEqual(response.status_code, 204)
```

Comment:
If we can now leave the user, organization-less, maybe worth changing the test name here?